Sources/MapboxNavigation/NavigationView.swift

```diff
@@ -125,10 +118,6 @@ open class NavigationView: UIView {
         return stackView
     }()
     
-    lazy var overviewButton = FloatingButton.rounded(image: Images.overview)
```

Comment:
IMHO these floating buttons should not be in `NavigationView` anymore as it can be used not only in active navigation anymore.

Reply:
Should we then move its management (showing, hiding, etc.) to the view controller? Seems like an extra knowledge for the view, if it doesn't contain the buttons.

Also, could it be a breaking change for the users of `NavigationView`?

Reply:
I do not consider this as a breaking change because all components are internal and not exposed to the user.

Reply:
There is actually another `ResumeButton` that currently resides in `NavigationView`, I haven't made any changes to it in scope of this PR, but it'd make sense to move it outside of `NavigationView` as well, because it's only active-navigation related UI component, and `NavigationView` is meant to be used in free-drive as well. 

The reason I keep it here is because it's likely to be removed/deprecated altogether to match Android behavior.

Reply:
Updated style:

<img width="66" alt="Screen Shot 2022-09-06 at 2 51 43 PM" src="https://user-images.githubusercontent.com/1496498/189418566-0d5bc608-1416-4aa7-b2af-68d44ad29b4f.png">

<img width="51" alt="Screen Shot 2022-09-06 at 3 00 31 PM" src="https://user-images.githubusercontent.com/1496498/189418569-1111fde3-180a-414a-847a-4c5a3c2ee886.png">